frontend/helpers/syntax-helper.tsx

```diff
@@ -0,0 +1,16 @@
+export default class SyntaxHelper {
```

Comment:
ya can we use these more in css than javascript ?

Reply:
I'm not sure how one would use css for these functions, and the values they return are used programmatically, not just in browser display 

If you can specify a little more what you're suggesting I'm happy to give it a try though!

Reply:
These seem like extra methods for something? the CapitalizeFirstChar can be done with `text-transform: capitalize;` but what are these js string manipulating functions for?

Reply:
It's so we can pass a single prop of the camel case input name (ie `firstName`) maintain that for the different IDs we need but still display the input label as in the wireframes (ie `First name:`)

And since 1) we need those IDs programmatically 2) we want the displayed label to correlate to those IDs dynamically (so that the component is reusable) and 3) it's not just capitalizing/de-capitalizing the letters, but also manipulating spacing - I don't think we can use CSS

Reply:
`getCamelCase` though actually is for an earlier rendition of the ticket & is no longer being used - I kept it around as I imagined it could be useful later, but I'm happy to delete that one if that helps address the concern!